jaeger-crossdock/https-proxy/proxy.template.conf

```diff
@@ -0,0 +1,20 @@
+server {
+    listen 8080;
+    server_name ${SERVER};
+    location / {
+        root /usr/share/nginx/html/;
+				index index.html;
+    }
+}
+
+server {
+    listen 14443 ssl;
+    server_name ${SERVER};
+
+    ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
```

Comment:
I know this is only for testing, but could we have a production-grade configuration here? Like, a secure set of ciphers and options like `ssl_prefer_server_ciphers on`. This way, we'd be testing with production settings, making sure that all involved components are up to modern standards.

For reference, this is what I used to have on a few servers of mine, which was "current" a couple of years ago:
```nginx
server {
	listen [::]:443 ssl http2;
	listen 443 ssl http2;
	server_name acme.example.com;
	ssl_certificate /etc/pki/tls/certs/acme.example.com.bundle.cert;
	ssl_certificate_key /etc/pki/tls/private/acme.example.com.key;
	ssl_session_cache shared:SSL:10m;
	ssl_session_timeout 10m;
	ssl_protocols TLSv1 TLSv1.1 TLSv1.2; # probably needs an update to include TLSv1.3?
	ssl_prefer_server_ciphers on;
	ssl_ciphers "EECDH+ECDSA+AESGCM EECDH+aRSA+AESGCM EECDH+ECDSA+SHA384 EECDH+ECDSA+SHA256 EECDH+aRSA+SHA384 EECDH+aRSA+SHA256 EECDH !EDH+aRSA !RC4 !aNULL !eNULL !LOW !3DES !MD5 !EXP !PSK !SRP !DSS";
	add_header Strict-Transport-Security max-age=31536000;
	root /usr/share/nginx/html/acme.example.com;
}

```